Makefile

```diff
@@ -15,8 +15,28 @@ SRCFILES := *.go
 all: firectl
 
 firectl: $(SRCFILES)
-	go build
+	go build -o firectl
 
+release: $(SRCFILES)
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build \
+		    -a \
+		    -installsuffix cgo \
+		    -ldflags "-s" \
+		    -gcflags=all=-trimpath=${TRIMPATH} \
+		    -asmflags=all=-trimpath=${TRIMPATH} \
+		    -o firectl
+
+sandbox-release-build:
```

Comment:
Like in https://github.com/firecracker-microvm/firecracker-go-sdk/pull/87, a stamp file and picking a better image name would be advisable.